book/website/community-handbook/local-build.md

```diff
@@ -1,154 +1,177 @@
+```text
+===============================================================================
 
-   ```bash
-   git checkout mybranch
-   jupyter-book build .
-   ```
+Finished generating HTML for book.
+Your book's HTML pages are here:
+    _build/html/
+You can look at your book by opening this file in a browser:
+    _build/html/index.html
+Or paste this line directly into your browser bar:
+    file:///<path to repository>/book/website/_build/html/index.html
 
-Follow the link as before and you will see changes specific to that branch rendered.
+===============================================================================
+```
 
-### Clean up a recent build
+Open `index.html` in your web browser to look at your local build.
 
-When you test your edits by building the book multiple times, it is better to clean up the last build before generating a new one.
-You can either manually delete the `book/website/_build` folder every time, or run this command:
+### Other Targets
+
+There are other build targets you can use.
+The `strict` target is useful for debugging.
+It will make any warnings raise an error, but also continue the build.
+That way, all warnings should be presented to you as errors.
+Run the strict build with,
+
+```console
+$ make strict
+```
+
+You can build the book with user pathways with,
 
 ```console
-cd book/website
-jupyter-book clean .
+$ make pathways
 ```
 
-More details on this process can be read in [Jupyter Book's documentation](https://jupyterbook.org/en/stable/basics/build.html?highlight=clean#clean-your-books-generated-files).
+This is the build of the book which is deployed to the website.
 
+## Building Previews for Different Branches
 
-### Check external links in the book
+The build process will use the source files from whatever branch you have checked out.
+If you have just cloned the repository, that will be the `main` branch.
 
-When editing or reviewing this book locally, you can run the Sphinx link checker with Jupyter Book to check if the external links mentioned in the book are valid.
-To run the link checker, use the following command:
+To build another branch, for example a feature branch you are working on you first switch to that branch,
 
 ```console
-cd book/website
-jupyter-book build . --builder linkcheck
+$ git switch mybranch
+$ make book
 ```
 
-The link checker checks if the each link resolves and prints the status on your terminal so that you can check and resolve any incorrect links.
-Read more about this in [Jupyter Book's documentation](https://jupyterbook.org/en/stable/advanced/html.html?highlight=check%20external#check-external-links-in-your-book).
+## Clean Up After a Build
+
+The build process generates a lot of files.
+Clearing these files to force a build from scratch may reveal errors and warnings that wouldn't be raised otherwise.
 
-## Why did we recommend using (mini)conda?
+To remove the outputs of builds use the `clean` target,
```

Comment:
Is this analogous to deleting `_build` or does it do anything else, I think I've just deleted _build previously when building locally? just curious.

Reply:
Yes, pretty much. It uses `jupyter-book clean` which should be more portable, if you have changed the build directory for example.

Reply:
right makes sense